Tests/SwiftDocCTests/Checker/Checkers/NonInclusiveLanguageCheckerTests.swift

```diff
@@ -31,6 +31,40 @@ class NonInclusiveLanguageCheckerTests: XCTestCase {
         XCTAssertEqual(range.upperBound.column, 16)
     }
 
+    func testMatchTermWithSpaces() throws {
+        let source = """
+# A White  listed title
+# A Black    listed title
+# A White listed title
+"""
+        let document = Document(parsing: source)
+        var checker = NonInclusiveLanguageChecker(sourceFile: nil)
+        checker.visit(document)
+        XCTAssertEqual(checker.problems.count, 3)
+
+        let problem = try XCTUnwrap(checker.problems.first)
+        let range = try XCTUnwrap(problem.diagnostic.range)
+        XCTAssertEqual(range.lowerBound.line, 1)
+        XCTAssertEqual(range.lowerBound.column, 5)
+        XCTAssertEqual(range.upperBound.line, 1)
+        XCTAssertEqual(range.upperBound.column, 18)
+
+        let problemTwo = try XCTUnwrap(checker.problems[1])
+        let rangeTwo = try XCTUnwrap(problemTwo.diagnostic.range)
+        XCTAssertEqual(rangeTwo.lowerBound.line, 2)
+        XCTAssertEqual(rangeTwo.lowerBound.column, 5)
+        XCTAssertEqual(rangeTwo.upperBound.line, 2)
+        XCTAssertEqual(rangeTwo.upperBound.column, 20)
+
+        let problemThree = try XCTUnwrap(checker.problems[2])
+        let rangeThree = try XCTUnwrap(problemThree.diagnostic.range)
+        XCTAssertEqual(rangeThree.lowerBound.line, 3)
+        XCTAssertEqual(rangeThree.lowerBound.column, 5)
+        XCTAssertEqual(rangeThree.upperBound.line, 3)
+        XCTAssertEqual(rangeThree.upperBound.column, 17)
+
```

Comment:
Sorry for the nitpick here, but I think we should remove line 65 and update the year in the license header from `2021` to `2021-2022` in the 2 files.

Reply:
Thank you for pointing that out, i updated that.